spek-extensions/robolectric/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply from: "$rootDir/gradle/common/dependencies.gradle"
+apply from: "$rootDir/gradle/common/kotlin-jvm.gradle"
+apply from: "$rootDir/gradle/common/junit-jupiter.gradle"
+apply plugin: 'org.jetbrains.dokka'
+
+configurations {
+    provided
+}
+
+sourceSets {
+    main {
+        compileClasspath += configurations.provided
+        runtimeClasspath += configurations.provided
+    }
+    test {
+        compileClasspath += configurations.provided
+        runtimeClasspath += configurations.provided
+    }
+}
+
+dependencies {
+    compile project(':spek-dsl:jvm')
+
+    compile group: 'junit', name: 'junit', version: '4.12'
+    compile group: 'org.robolectric', name: 'robolectric', version: '3.4.2'
```

Comment:
Please use `'org.robolectric:robolectric:3.4.2'` form of dependency declaration for consistency

Reply:
Done.